DOCS/man/mpv.rst

```diff
@@ -1200,10 +1200,19 @@ PROTOCOLS
 
     Digital TV via DVB. (Linux only.)
 
-``mf://[filemask|@listfile]`` ``--mf-...``
+``mf://[@listfile|filemask|glob|printf-format]`` ``--mf-...``
 
     Play a series of images as video.
 
+    If the path begins with ``@``, it is interpreted as a sequence of paths
```

Comment:
> If the path begins with `@`, it is interpreted as a sequence of paths separated by newlines

this is still incorrect, see previous comment

Reply:
The part of a URL after the scheme is called path, I don't know how else you want to call it? For now I specificied URL path to differentiate it from file paths it contains.

Reply:
No, not that part.

> **it is interpreted as a sequence of paths separated by newlines**

If I use `mf://@foo` mpv will go and open `foo` and treat every line in it as a path to be played.
That's not what the documentation says.

Reply:
Ah I've been testing with `mf://@<(print -l *.jpg)` and totally forgot it's supposed to accept a file.